### PR TITLE
Add Japanese course and interactive tutorials

### PR DESCRIPTION
**Pull request by SuzukiKatsuma**
## What does this PR do?
Add resources

### Description
Add freeCodeCamp resources.

### Why is this valuable (or not)?
These are popular resources for learning programming through hands-on experience.

### How do we know it's really free?
These resources are made available by the nonprofit organization "freeCodeCamp".

### For book lists, is it a book? For course lists, is it a course? etc.
These are course and interactive tutorials.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ja.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 ### Index
 
+* [0 - 大規模公開オンライン講座(MOOC)](#0---%E5%A4%A7%E8%A6%8F%E6%A8%A1%E5%85%AC%E9%96%8B%E3%82%AA%E3%83%B3%E3%83%A9%E3%82%A4%E3%83%B3%E8%AC%9B%E5%BA%A7mooc)
 * [Scratch](#scratch)
 
 
+### 0 - 大規模公開オンライン講座(MOOC)
```
Comment (LuigiImVector) [inline courses/free-courses-ja.md:7]:
```suggestion
* [0 - 大規模公開オンライン講座 (MOOC)](#0---mooc)
* [Scratch](#scratch)


### <a id="0---mooc"></a>0 - 大規模公開オンライン講座 (MOOC)
```

easier to understand

**Review by LuigiImVector — APPROVED** [review]
LGTM

File: courses/free-courses-ja.md (hunk 1 above)
Comment (LuigiImVector) [reply courses/free-courses-ja.md]: You can mark this as resolved

Comment (SuzukiKatsuma) [reply courses/free-courses-ja.md]: Thank you for the review!

**Review by davorpa — APPROVED** [review]

**Pull request merged**